var/spack/repos/builtin/packages/sphexa/package.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class Sphexa(CMakePackage, CudaPackage, ROCmPackage):
+    """SPH and N-body simulation framework"""
+
+    homepage = "https://github.com/sphexa-org/sphexa"
+    url = "https://github.com/sphexa-org/sphexa/archive/v0.0.tar.gz"
+    git = "git@github.com:sphexa-org/sphexa.git"
+    maintainers = ["sekelle"]
+
+    license("MIT")
+
+    version("0.93.1", sha256="95a93d0063ac8857b9be12c1aca24f5b2eef9dd4ffe8cf3f6b552a4dd54b940f")
+    version("0.93", sha256="03bc24d5fc2ca8f42f18c02af39dd10a36fb4118f4d5cb5d6c651f9b7a2bece6")
+    version("0.92", sha256="cb38a6e11c72ec0c40e8a28a9b0725c8b596d274d3164ab8dffa73501db0e3aa")
+    version("0.91", sha256="2811ac0d3a70847219c43b2afce2505cee88575a4da74915937b1b3f0ee43df3")
+    version("0.82", sha256="23ef1057996e1b5280b1dca88730efa398a830a42715421688bbdc81d8d090fe")
+    version("0.81", sha256="a0f90207d78937db54e23b002e911be7138abc0f080fe2a529350582fa4e0e92")
+    version("0.8", sha256="cecc0076d7b9a68fa79d3985b46ed67c49846f26df66eb98db956abcecb46b7c")
```

Comment:
```suggestion
    version("0.93", sha256="b4ffd696a4981c7396c7ee211d6a815e405e2345417903e4d8616508866c96e2")
    version("0.92", sha256="3a9b26985999e5b05750af1e01fed6ae8cf9f7b5f7b6cbcd3079d4c10b1167df")
    version("0.91", sha256="948fbd6557615944d0438682e1ef03d37651ae4462e758df08c85f861e0eba8b")
    version("0.90", sha256="e3974c7f0b9e2e47cc92edac8346375276377ac9c14356e3b286fdea7096f699")
    version("0.82", sha256="85fcb4e784eff2eb1aa019c8b31c0c372bd3ebc4ec37b229ba85d54f565f457c")
    version("0.81", sha256="8da971eba2d34b9b3e6fedc050365f9a16fbd5c319414a160a0fe2c31d80f11b")
    version("0.8", sha256="012beaddf5181197cd69e1d5498b06f51b9946e3ddacd3d1522b27d034641776")
```
`spack checksum sphexa` gives me different checksums for version others than `0.93.1` and `spack install --until=cmake -v sphexa@=0.93` indeed fails with `ChecksumError`.

Reply:
Strange, no idea how the older versions got the wrong hash. I've decided to remove them as it's not worth adding the extra logic to the package to support the older versions correctly, i.e. supporting older CMake options not named in the conventional way.

Reply:
Sounds good.